src/util/config.rs

```diff
@@ -518,7 +525,7 @@ impl<'de> Deserialize<'de> for ReadableDuration {
                     if unit >= last_unit {
                         return Err(E::invalid_value(
                             Unexpected::Str(dur_str),
-                            &"h, m, s, ms should occur in giving order.",
+                            &"d, h, m, s, ms should occur in giving order.",
```

Comment:
I consider it a legacy typo. Should we change "giving" into "given"? @tikv/pingcap-i18n

Reply:
@ice1000 Yes. I think we should change it to "given".

Reply:
May i change it in this PR? @lilin90 @ice1000

Reply:
I think it's fine. Do it.

Reply:
I think it's fine. Ping @lilin90 @overvenus